### ISSUE TITLE
[Bug report] local data not persisting after refresh

### ISSUE THREAD

**Issue by creeperita09** [riccardocalligaro/registro_elettronico#63]
Hi, all of the local data on the latest version does not persist After e refresh or closing and reopening the app, i tried using the report a bug button but It does nothing.

**Comment by creeperita09** [comment]
Also i forgot, the credentials that are automatically inputed in the web login page do not seen to be correct (maybe it's inputing the pw web formatted (i have a # in the pw)) And the background in the didactic materials does not reflect the One in the theme.

**Comment by riccardocalligaro** [comment]
hello, thanks for the report. you mean all the local data like downloaded lessons/grades/etc? or the data you manually enter in the db like local grades and local events?

**Comment by riccardocalligaro** [comment]
the web login bug has been like this for a while I think. but the data persistency issue seems a major one. I don't have android anymore and I asked my friends to try it and on Android 13 and 14 it seemed to work fine as usual

**Comment by creeperita09** [comment]
The data manually inputed

**Comment by creeperita09** [comment]
Oh and the statistics Page seems to not be working

**Comment by riccardocalligaro** [comment]
> Oh and the statistics Page seems to not be working

yes I received a lot of reports for this but every time I try the statistics page I don't get any issue. Maybe it's only on release mode. I will check

**Comment by creeperita09** [comment]
I really like this project, i'll learn flutter and i'll help you out

**Comment by creeperita09** [comment]
> > Oh and the statistics Page seems to not be working
> 
> yes I received a lot of reports for this but every time I try the statistics page I don't get any issue. Maybe it's only on release mode. I will check

You could try an Android emulator

**Comment by creeperita09** [comment]
hmm apparently the issue with the statistics page solved itself, it is now working, i'll check if the user imported data persists
